src/views/layouts/master.twig

```diff
@@ -120,8 +120,27 @@
         window.header_is_sticky = "{{theme.settings.get('header_is_sticky', 'Default Value')}}"
         window.imageZoom = "{{theme.settings.get('imageZoom')}}"
     </script>
-    
-    {% hook 'head:start' %}
+{#{% hook 'head:start' %}#}
+<script data-cfasync="false">
+  window.salla = window.Salla = window.Salla || {};
+  Salla.onReady =
+  	Salla.onReady ||
+  	function (callback) {
+  		return new Promise(function (resolve) {
+  			return Salla.status == "ready"
+  				? resolve(callback ? callback(salla.config.all()) : salla.config.all())
+  				: window.addEventListener(
+  						"twilight::initiated",
+  						function (config) {
+  							resolve(callback ? callback(config.detail) : config.detail);
+  						},
+  						{ once: true },
+  					);
+  		});
+  	};
+</script>
+
+<script async src="https://cdn.salla.network/js/twilight/tests/featureSTORE-1611-enhanced-review-system/c0057cf0b2b1603dee0b32f2a883166cbd051a30/twilight.js"></script>
```

Comment:
i dont think that this path is normal !

Reply:
@egkrinkel Take a look at the PR title 👀